DOCS-8272 OAS 3 (#150)
mulesoft/docs-api-community-manager: modules/ROOT/pages/update-acm.adoc

To update your API community:

> . Ensure your Salesforce organizations have the "External Data Change Tracking for OData 4.0 Adapter" feature enabled.

Reviewer: ensure > ensure that
```suggestion
. Ensure that your Salesforce organizations have the External Data Change Tracking for OData 4.0 Adapter feature enabled:
```
I think since it is all init. cap. it likely does not need quotation marks.

Author: Fixed.

> .. Before upgrading the package in your production or sandbox organization, check that your organization has this feature enabled by going to *Setup* > *External Data Source* and editing the Exchange data source.

Reviewer: production or sandbox environment? rather than organization?

Author: In this case org is correct. Each env has an org, and packages are upgraded in orgs.

> .. If you don't see a checkbox field with the label *Eligible for External Change Data Capture*, raise a support case through the MuleSoft support portal, and include the organization IDs of all your Salesforce API Community Manager organizations, including production and all sandboxes.

Reviewer: ```suggestion
.. If you don't see an *Eligible for External Change Data Capture* checkbox, raise a support case through the MuleSoft support portal and list the IDs of all your Salesforce API Community Manager organizations, including production and all sandboxes.
```
"including the IDs of all production and sandbox environments"?

Author: Fixed.

+

> An attempt to upgrade the package without this feature enabled causes the process to fail with errors due to missing objects and fields related to a missing "ChangeEvent" object.

Reviewer: ```suggestion
Attempting to upgrade the package without this feature enabled causes the process to fail with errors due to missing objects and fields related to a missing ChangeEvent object.
```

Author: Fixed.
